packages/ocaml_intrinsics/ocaml_intrinsics.v0.15.0/opam

```diff
@@ -14,7 +14,7 @@ depends: [
   "dune"              {>= "2.0.0"}
   "dune-configurator"
 ]
-available: (arch = "x86_32" | arch = "x86_64") & os != "win32" & os != "macos"
+available: (arch = "x86_32" | arch = "x86_64") & os != "win32"
```

Comment:
this also constraints newer macs out (arm64)

Reply:
> this also constraints newer macs out (arm64)

indeed, this fix doesn't address the problem on the new macos on arm targets.